File: docs/components_page/components/card/list_group.R
library(dashBootstrapComponents)


card <- dbcCard(
  dbcListGroup(
    list(
      dbcListGroupItem("Item 1"),
      dbcListGroupItem("Item 2"),
      dbcListGroupItem("Item 3")
    ),
    flush = TRUE
  ),
  style = list(width = "18rem")
)

File: docs/components_page/components/card/list_group.jl
using DashBootstrapComponents

card = dbc_card(
    dbc_listgroup(
        [
            dbc_listgroupitem("Item 1"),
            dbc_listgroupitem("Item 2"),
            dbc_listgroupitem("Item 3"),
        ],
        flush = true,
    ),
    style = Dict("width" => "18rem"),
);
